### PR TITLE
Uprade to 1.1 and fixes Twisted by retrograding it

### PR DESCRIPTION
**Pull request by Thovi98**
- Upgrade to 1.1

- Fixes Twisted by retrograding it (cf. https://forum.yunohost.org/t/funkwhale-server-crash/14774/2)

Tested install, but not upgrade.

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: sources/extra_files/app/api/requirements/base.txt [hunk 1]
```diff
@@ -71,4 +73,4 @@ service_identity~=18.1
 markdown~=3.2
 bleach~=3.1
 feedparser==6.0.0b3
-watchdog~=0.10
\ No newline at end of file
+watchdog~=0.10
```
Comment (Sporiff) [inline sources/extra_files/app/api/requirements/base.txt:76]: I think this needs to be bumped to 1.0.2 in line with upstream

**Review by Sporiff — CHANGES_REQUESTED** [review]
In the base requirements, we should also replace line 52 with `git+https://github.com/agateblue/requests-http-signature.git@signature-header-support` to reflect the location change. In theory it should automatically resolve but it's better to be neat about it. As long as everything aligns with [upstream](https://dev.funkwhale.audio/funkwhale/funkwhale/-/blob/master/api/requirements/base.txt) this should be fine

Apart from that and my comment on the watchdog line this looks fine to me. I don't have time right now to test this (currently in the middle of moving country) but if somebody wants to give it a try that would be great 👍

**Comment by Thovi98** [comment]
I’ve copy pasted the upstream file, and checked, now everything is fine I think. 

I had tested it already, works great !

**Comment by ericgaspar** [comment]
Is this PR ready to be merged? 👀

**Comment by Thovi98** [comment]
From my side, it is, and @Sporiff approuved, so I think it’s okay

**Comment by ericgaspar** [comment]
Merging as the app is level 0. 👍🏻

**Pull request merged**